azavea/green-equity-demo
# Convert per capita map to choropleth

## Overview

This PR converts the per-capita map to a choropleth for better readability. It also does a few organizational things:
- Moves maps and their components to folders
- Moves per-capita data fetching to a hook in a separate file
- Stores tooltip divs in state to facilitate separation of tooltip creation and render

Closes #65 

### Demo

<img width="1064" alt="Screenshot 2023-03-10 at 10 21 07 AM" src="https://user-images.githubusercontent.com/8356789/224368469-9dfd1389-a913-4428-b4c4-e9e9fe142b8b.png">

Mobile

<img width="477" alt="Screenshot 2023-03-10 at 10 23 49 AM" src="https://user-images.githubusercontent.com/8356789/224369063-1f7b73f9-bfb7-4e8a-82c2-eafcad548a47.png">


### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

- http://localhost:8765/
- Ensure per-capita map is choropleth and looks good
- Squeeze window to enable mobile mode and ensure still looks good

 ## Checklist
